### ISSUE TITLE
`aquest train` not documented well

### ISSUE THREAD

**Issue by prophnoran** [prophecymud/prophecy-issues#22]
At Floyd:

```> aq
Usage: aquest <subcommand>, where subcommand is one of:

  start
  request
  complete
  info
  time
  deny
  points
  list
  buy
  practice
  gain
  unflag
  destring
  enchant
  disenchant
```

Note that `aq train` is not listed. It is not listed in https://github.com/prophecymud/helps/blob/master/wikihelp/Aquest . It is listed in `help aq_commands2` (but `help aquest` doesn't point you there) with this text:

```
train    - Allows you to increase one of your stats by one point, eg
           'Aquest train str' will improve your strength. You can't train your
           stats over their allowable maximum.
```

However this does not tell me how much it will cost. And trying the command out:

```
 aq train
You don't have enough aqps to train.
```

also does not tell me how much it will cost (this character has 657, so must be expensive!)

To summarize, there are three issues:

1) `aquest` by itself doesn't list all the possible subcommands, and
2) there's no way to find out (that I can find) how much an `aq train` will cost.
3) the `help aq_commands` help page is not very discoverable

**Comment by anhegofcherek** [comment]
* I've improved the help you get when you `aquest` or `aquest help` to include `train` (and `sell`) and argument shape and some brief explanation
* `aquest list` is the canonical price list, but I appreciate that's not hugely discoverable and needs some updates itself - but the above improvement should help somewhat

I'd like to tackle the help aspect separately, but hopefully this should hit the core stuff! (Will land in the next deploy)
